### PR TITLE
update system requirements & pre-release disclaimer

### PR DESCRIPTION
**Pull request by xu-yuan1**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: docs/install_guides/pypi-openvino-rt.md [hunk 1]
```diff
@@ -15,6 +18,11 @@ Before you start the installation, check the supported operating systems and req
 |   macOS* 10.15.x versions                                    | 3.6, 3.7, 3.8, 3.9                                  |
 |   Windows 10*, 64-bit                                        | 3.6, 3.7, 3.8, 3.9                                  |
 
+**C++ libraries** are also required for the installation. To install C++ libraries:
+- On Windows*, you can [download the Visual Studio Redistributable file (.exe)](https://aka.ms/vs/17/release/vc_redist.x64.exe).
+- On macOS*, you can use `xcode-select --install`.
+- On Linux*, you can use `sudo apt install build-essential`.
```
Comment (ilya-lavrenov) [inline docs/install_guides/pypi-openvino-rt.md:24]: really? who said that we need this on OSX and Linux?

Comment (ilya-lavrenov) [reply docs/install_guides/pypi-openvino-rt.md:24]: about Windows, we already state it below "On Windows*, some libraries are necessary to run OpenVINO. To resolve this issue, install the [C++ redistributable (.exe)](https://aka.ms/vs/17/release/vc_redist.x64.exe"

Comment (xu-yuan1) [reply docs/install_guides/pypi-openvino-rt.md:24]: > really? who said that we need this on OSX and Linux?

It's suggested by Helena, and Zoya from validation has tested them.

Comment (xu-yuan1) [reply docs/install_guides/pypi-openvino-rt.md:24]: > about Windows, we already state it below "On Windows*, some libraries are necessary to run OpenVINO. To resolve this issue, install the [C++ redistributable (.exe)](https://aka.ms/vs/17/release/vc_redist.x64.exe"

Well, it's in the Troubleshooting part, so it's not easy for users to find until they have met issues. It makes more sense to be in System Requirements.

File: docs/install_guides/pypi-openvino-dev.md [hunk 2]
```diff
@@ -117,15 +126,15 @@ For example, to install and configure the components for working with TensorFlow
...
+| [Model Optimizer](https://docs.openvino.ai/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) | `mo` |**Model Optimizer** imports, converts, and optimizes models that were trained in popular frameworks to a format usable by OpenVINO components. <br>Supported frameworks include Caffe\*, TensorFlow\*, MXNet\*, PaddlePaddle\*, and ONNX\*.                                               |
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/pypi-openvino-dev.md:129]: When we create an official release branch, the version will need to be changed to that release version too. So here the version is most probably for pre-releases. Do we decide to use the nightly (most up-to-date) version for pre-releases? If yes, I can change it to nightly.

**Review by artanokhov — APPROVED** [review]

File: docs/install_guides/pypi-openvino-rt.md (hunk 1 above)
Comment (ilya-lavrenov) [reply docs/install_guides/pypi-openvino-rt.md:24]: > > really? who said that we need this on OSX and Linux?
> 
> It's suggested by Helena, and Zoya from validation has tested them.

I need proofs that w/o these commands python does not work. I've also tested python packages on empty ubuntu 20.04 docker image and I don't build-essentials. If we can think a bit, we can understand that for already **built** package we don't need build tools or build-essential

File: docs/install_guides/pypi-openvino-dev.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ilya-lavrenov) [reply docs/install_guides/pypi-openvino-dev.md:129]: here I see that we use nightly for pre-release https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/wheel/setup.py#L485

[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/pypi-openvino-dev.md]: OK, updated.

File: docs/install_guides/pypi-openvino-rt.md (hunk 1 above)
Comment (ilya-lavrenov) [reply docs/install_guides/pypi-openvino-rt.md:24]: can we remove steps for OSX and Linux?

Comment (xu-yuan1) [reply docs/install_guides/pypi-openvino-rt.md]: > can we remove steps for OSX and Linux?

Removed. :)

**Review by ilya-lavrenov — APPROVED** [review]

**Review by kblaszczak-intel — APPROVED** [review]

**Pull request merged**